experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -346,6 +349,118 @@ describe("SequenceField - Rebaser Axioms", () => {
 	});
 });
 
+interface TestState {
+	length: number;
+	numNodes: number[];
+}
+
+type SequenceFieldTestState = FieldStateTree<TestState, TestChangeset>;
+
+/**
+ * See {@link ChildStateGenerator}
+ */
+const generateChildStates: ChildStateGenerator<TestState, TestChangeset> = function* (
+	state: SequenceFieldTestState,
+	tagFromIntention: (intention: number) => RevisionTag,
+	mintIntention: () => number,
+): Iterable<SequenceFieldTestState> {
+	const numNodes = state.content.numNodes;
+
+	// Undo the most recent edit
+	if (state.mostRecentEdit !== undefined) {
+		assert(state.parent?.content !== undefined, "Must have parent state to undo");
+		const undoIntention = mintIntention();
+		const invertedEdit = invert(state.mostRecentEdit.changeset);
+		yield {
+			content: state.parent.content,
+			mostRecentEdit: {
+				changeset: tagChange(invertedEdit, tagFromIntention(undoIntention)),
+				intention: undoIntention,
+				description: `Undo:${state.mostRecentEdit.description}`,
+			},
+			parent: state,
+		};
+	}
+
+	for (const nodeCount of numNodes) {
+		// MoveIn nodeCount nodes
+		const moveInIntention = mintIntention();
+		yield {
+			content: state.content,
+			mostRecentEdit: {
+				changeset: tagChange(
+					Change.move(1, nodeCount, 0),
+					tagFromIntention(moveInIntention),
+				),
+				intention: moveInIntention,
+				description: `MoveIn${nodeCount}${nodeCount === 1 ? "Node" : "Nodes"}From1To0`,
+			},
+			parent: state,
+		};
+
+		// MoveOut nodeCount nodes
+		const moveOutIntention = mintIntention();
+		yield {
+			content: state.content,
+			mostRecentEdit: {
+				changeset: tagChange(
+					Change.move(0, nodeCount, 1),
+					tagFromIntention(moveOutIntention),
+				),
+				intention: moveOutIntention,
+				description: `MoveOut${nodeCount}${nodeCount === 1 ? "Node" : "Nodes"}From0To1`,
+			},
+			parent: state,
+		};
+		for (let i = 0; i <= state.content.length; i++) {
```

Comment:
Perhaps instead of generating all possible inserts we could go from 0 to <= min(K, length).
Where `K` would be a configurable value. I suggest starting with 2 since that will allow us to generate (when considering concurrent changes) the four sitatuions that matter:
* before a node that isn't being affected by a concurrent change 
* before a node that is being affected by a concurrent change
* after a node that is being affected by a concurrent change
* after a node that isn't being affected by a concurrent change

Should reduce the branching factor a bit

Reply:
This is now part of the configuration for the suite (maxIndex)